src/Chapter09.Tests/Listing09.14.RecordDeconstructors.Tests.cs

```diff
@@ -0,0 +1,24 @@
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_14.Tests;
+
+using AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_14;
+
+
+[TestClass]
+public class DeconstructorTest
+{
+    [TestMethod]
+    public void DeconstructorValues_MatchAngleValues()
+    {
+        (int degrees, int minutes, int seconds) = (
+            90, 0, 0);
+
+        // The constructor is generated using positional parameters
+        Angle angle = new(degrees, minutes, seconds);
+
+        angle.Deconstruct(out int d, out int m, out int s);
```

Comment:
nit: We generally only use abbreviations when they are obvious.
```suggestion
        angle.Deconstruct(out int degrees, out int minutes, out int seconds);
```